code/index.html

```diff
@@ -4,34 +4,108 @@
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="stylesheet" href="./normalize.css" />
     <title>Technigo Project Cluedo</title>
   </head>
   <body>
+    <!-- BACKGROUND -->
+    <div class="background">
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+    </div>
+    <!-- SOURCE for Background idea, which was modified from the original: https://dev.to/vincenius/websites-for-creative-backgrounds-1mao -->
+    <div class="title">
+      <h1>CLUEDO</h1>
+    </div>
+    <div class="title-p">
+      <p>
+        Choose your suspect, the room and a weapon to reveal the crime.
+        <br />You may shuffle the cards as many times as you'd like to, but the
+        mystery won't be revealed unless all three decks are shuffled! When
+        you're finished, click on the 'New Round' button.
+      </p>
+    </div>
     <section class="cards">
       <article class="killer-deck" id="killerCard">
-        <p>KILLER</p>
+        <p>SUSPECT</p>
         <p id="killerName"></p>
         <p id="killerAge"></p>
         <p id="killerOccupation"></p>
-        <img id="killerImage" src="" height="100" />
+        <img id="killerImage" src="" height="150" />
         <p id="killerDescription"></p>
+        <p id="favouriteWeapon"></p>
+      </article>
+      <article class="room-deck" id="roomCard">
+        <p>ROOM</p>
+        <p id="roomName"></p>
       </article>
       <article class="weapon-deck" id="weaponCard">
         <p>WEAPON</p>
         <p id="weaponName"></p>
+        <p id="weaponFact"></p>
         <p id="weaponWeight"></p>
       </article>
-      <article class="room-deck" id="roomCard">
-        <p>ROOM</p>
-        <p id="roomName"></p>
-      </article>
     </section>
 
     <h1 id="mystery"></h1>
 
-    <button>REVEAL THE CRIME</button>
+    <button id="crimeButton">REVEAL THE MYSTERY</button>
 
-    <script src="clue.js"></script>
+    <div class="refresh">
+      <button id="refreshPage" onclick="window.location.reload();">
```

Comment:
Smart that you used the refresh function directly in html, instead of creating a function in JS. I will do the same next time. 😎

Reply:
Maybe this button doesn't have to be there all the time though ;)